tests/unittests/storage/storage6/storage6.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using NUnit.Framework;
+using Trinity;
+using Trinity.Storage;
+
+namespace storage6
+{
+    public class storage6
+    {
+		public static void StorageSavedHandler()
+        {
+            Console.WriteLine("Save Finished.");
+            return;
+        }
+        public static void StorageBeforeSaveHandler()
+        {
+            Console.WriteLine("Save Started.");
+            return;
+        }
+        public static void StorageLoadedHandler()
+        {
+            Console.WriteLine("Load Finished.");
+            return;
+        }
+        public static void StorageBeforeLoadHandler()
+        {
+            Console.WriteLine("Load Started.");
+            return;
+        }
+        public static void StorageResetHandler()
+        {
+            Console.WriteLine("Reset Finished.");
+            return;
+        }
+        public static void StorageBeforeResetHandler()
+        {
+            Console.WriteLine("Reset Started.");
+            return;
+        }
+		
+		[Test]
+        public unsafe void T1()
+        {
+            Global.LocalStorage.StorageBeforeLoad += new Action(StorageBeforeLoadHandler);
+            Global.LocalStorage.StorageLoaded += new Action(StorageLoadedHandler);
+            Global.LocalStorage.LoadStorage();
```

Comment:
In this case, you can simply add some bool variables and initialize them to `false`. Everytime a hook is triggered you flip one of them true.

Then you should be able to make assertions based on these values. Pay attention to the order.